tests/fourier/test_qnode_spectrum.py

```diff
@@ -20,7 +20,6 @@
 import pennylane as qml
 from pennylane import numpy as pnp
 from pennylane.fourier.qnode_spectrum import qnode_spectrum, _process_ids
-from pennylane.transforms import classical_jacobian
```

Comment:
was this never being used?

Reply:
From what I could tell it is unused in the module, I think I saw a few instances of this.